src/realm/sync/client.cpp

```diff
@@ -1,5 +1,6 @@
 #include <tuple>
 #include <atomic>
+#include <iostream>
```

Comment:
is this left over from debugging something?

Reply:
removed